### PR TITLE
Update documentation and communications

### PR DESCRIPTION
**Pull request by ebarojas**
Update README and communication to users in landing and other places, update stale links, etc.

### CODE REVIEW

**Comment by ebarojas** [comment]
Need to merge to Xtornasol's current work in progress.

File: rexchain/templates/base.html [hunk 1]
```diff
@@ -93,27 +92,30 @@
           <div class="row">
             <div id="menu1" class="footer_menu col-xs-6">
               <ul class="list-unstyled">
-                <li class="footeranch"><a id="gotoblog" href="http://blog.prescrypto.com">Blog</a></li>
-                <li class="footeranch"><a id="gotocontactfooter" href="mailto:hola@prescrypto.com?Subject=RexChain-Contact">Contacto</a></li>
+                <li class="footeranch"><a id="gotoblog" href="https://blog.prescrypto.com">Blog</a></li>
+                <li class="footeranch"><a id="gotocontactfooter" href="https://www.prescrypto.com/contact">Contacto</a></li>
```
Comment (xtornasol512) [inline rexchain/templates/base.html:96]: TODO add some utm params

Comment (ebarojas) [reply rexchain/templates/base.html:96]: Todo here: https://github.com/Prescrypto/RexChain/issues/67

File: rexchain/templates/blockchain/rx_detail.html [hunk 2]
```diff
@@ -14,14 +14,13 @@ <h2 style="color:#000; margin-left:4em;font-weight:700;">Detalle de la Transacci
       <div class="col-xs-12 col-lg-8 col-lg-offset-2">
         <div class="row">
           <article class="col-xs-12 col-lg-12">
-
-            {% if rx.transaction.block %}
```
Comment (xtornasol512) [inline rexchain/templates/blockchain/rx_detail.html:18]: Estuvo bien quitarlo

**Review by xtornasol512 — APPROVED** [review]
Lo veo bien!

**Pull request merged**